src/ansys/fluent/core/services/solution_variables.py

```diff
@@ -430,61 +476,67 @@ def __getitem__(self, name):
 
     def __init__(
         self,
-        service: SVARService,
-        svar_info: SVARInfo,
+        service: SolutionVariableService,
+        solution_variable_info: SolutionVariableInfo,
     ):
         self._service = service
-        self._svar_info = svar_info
+        self._solution_variable_info = solution_variable_info
 
-    def _update_svar_info(self):
-        self._allowed_zone_names = _AllowedZoneNames(self._svar_info)
+    def _update_solution_variable_info(self):
+        self._allowed_zone_names = _AllowedZoneNames(self._solution_variable_info)
 
-        self._allowed_domain_names = _AllowedDomainNames(self._svar_info)
+        self._allowed_domain_names = _AllowedDomainNames(self._solution_variable_info)
 
-        self._allowed_svar_names = _AllowedSvarNames(self._svar_info)
+        self._allowed_solution_variable_names = _AllowedSvarNames(
+            self._solution_variable_info
+        )
         svar_args = dict(
-            zone_names=self._allowed_zone_names, svar_name=self._allowed_svar_names
+            zone_names=self._allowed_zone_names,
+            solution_variable_name=self._allowed_solution_variable_names,
         )
 
-        self.get_svar_data = override_help_text(
+        self.get_solution_variable_data = override_help_text(
             _SvarMethod(
-                svar_accessor=self.get_svar_data,
+                svar_accessor=self.get_solution_variable_data,
                 args_allowed_values_accessors=svar_args,
             ),
-            SVARData.get_svar_data,
+            SolutionVariableData.get_solution_variable_data,
         )
 
     def get_array(
```

Comment:
Also we should reame this to `empty()` or `create_empty_array()`. In one defect there was a confusion whether this is returned by solver.

Reply:
Right `get_array()` returns an empty array.

![image](https://github.com/ansys/pyfluent/assets/106588300/6a1d2886-1124-489b-b7e7-6074bdc4a2dd)

Reply:
Done. Thank you.